src/components/field-errors/field-errors.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import ErrorMessage from '../messages/error-message';
+import messages from './messages';
+
+const isObject = obj => typeof obj === 'object';
```

Comment:
Feels like this could as well be `const hasErrors = errors => typeof errors !== 'object' || Object.keys(errors).length <= 0`. Doesn't matter though I guess.